components/probe-card/index.tsx

```diff
@@ -26,6 +26,12 @@ const ProbeCard: FunctionComponent<ProbeCardProps> = ({ probe, id }) => {
     handleRemoveProbe,
   } = useContext(ProbeContext);
 
+  const isAlertSelected = (alert: string) => {
+    const exists = probe.alerts.filter((item: string) => item.includes(alert));
```

Comment:
for safety, there may be uploaded json with no alerts key

```suggestion
    const exists = probe?.alerts?.filter((item: string) => item.includes(alert));
```

Reply:
Isn't it handled by `contexts/importer.ts` already? 

```js
if (!item.alerts)
  return {
    ...item,
    alerts: ['status-not-2xx', 'response-time-greater-than-2000-ms'],
  };
```

When you uploaded the JSON, it detects that if you don't have any alerts it will automatically set the default alerts

Reply:
i see, you are right